Content.Server/GameObjects/Components/HeadsetComponent.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Content.Server.GameObjects.EntitySystems;
+using Content.Server.Interfaces;
+using Content.Shared.Chat;
+using Content.Shared.GameObjects.Components.Items;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using NFluidsynth;
+using Robust.Server.Interfaces.GameObjects;
+using Robust.Server.Interfaces.Player;
+using Robust.Shared.Containers;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.GameObjects.Components;
+using Robust.Shared.Interfaces.Network;
+using Robust.Shared.IoC;
+using Robust.Shared.Map;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Content.Server.GameObjects.Components
+{
+    [RegisterComponent]
+    [ComponentReference(typeof(IRadio))]
+    [ComponentReference(typeof(IListen))]
+    public class HeadsetComponent : Component, IListen, IRadio
+    {
+#pragma warning disable 649
+        [Dependency] private readonly IServerNetManager _netManager;
+        [Dependency] private readonly IEntitySystemManager _entitySystemManager;
+#pragma warning restore 649
+
+        public override string Name => "Headset";
+
+        //only listens to speech in exact same position
+        private int _listenRange = 0;
```

Comment:
Some food for thought as often I forget; you can add the [ViewVariables] attribute to properties (i.e. the ones with either => or {get; set;} on them) and view these in game via the vv tool. This can make playtesting a lot easier as you don't need to re-compile to change a variable.